src/main/kotlin/no/nav/integrasjon/ldap/LDAPGroup.kt

```diff
@@ -358,11 +358,13 @@ class LDAPGroup(private val config: FasitProperties) :
 
 /**
  * Enum class KafkaGroupType with LDAP group prefix included
- * Each topic has 2 groups
+ * Each topic has 3 groups
  * - a producer group with members allowed to produce events to topic
  * - a consumer group with members allowed to consume events from topic
+ * - a connect group with members allowed to create on topic
  */
 enum class KafkaGroupType(val prefix: String) {
+    @SerializedName("CONNECT") CONNECT("CO-"),
```

Comment:
Burde kanskje renames til CREATE og ha prefix KC for Kafka Create

Reply:
Kan godt kalle den `CREATE`, men `KC` er brukt av `CONSUMER`. Ev. kan vi legge `CREATE` på `PRODUCER`, slik som `kafka-acls`-verktøyet gjør.